platformio.ini

```diff
@@ -1371,6 +1371,28 @@ extra_scripts        = ${common.extra_scripts}
   buildroot/share/PlatformIO/scripts/stm32_bootloader.py
   buildroot/share/PlatformIO/scripts/mks_encrypt.py
 
+#
+# MKS Robin Nano V3
+#
+[env:mks_robin_nano_v3]
+platform      = ${common_stm32.platform}
+extends       = common_stm32
+build_flags   = ${common_stm32.build_flags} -DHAL_PCD_MODULE_ENABLED -DUSBCON -DUSBD_USE_CDC
```

Comment:
`USBCON` and `USBD_USE_CDC` are already in `common_stm32`.

You probably don't need HAL_PCD_MODULE_ENABLED either...
Some board set it and others do not. Maybe they are set differently in hal_conf_extra.

Reply:
Yeah, I know. But I plan to remove that USB_USE_CDC and USBCON from base, as more boards with hw serial migrate from stm32f1. 
It it will change a bit with the upcoming changes to use MSC. In this commit I just let it.

Reply:
I let `HAL_PCD_MODULE_ENABLED` because the final version would be more like this: 
`
build_flags   = ${common_stm32.build_flags} -DHAL_PCD_MODULE_ENABLED -DUSBCON -DUSBHOST -DHAL_HCD_MODULE_ENABLED -DUSE_USBHOST_HS -DUSE_USB_HS_IN_FS -DUSBD_USE_CDC_MSC
`